zathura/utils.c

```diff
@@ -257,3 +257,26 @@ GdkAtom* get_selection(zathura_t* zathura)
 
   return selection;
 }
+
+unsigned int find_first_page_column(const char* first_page_column_list,
+                                    const unsigned int pages_per_row) {
+  /* copy settings list */
+  char* settings = malloc(strlen(first_page_column_list));
+  strcpy(settings, first_page_column_list);
+
+  /* save first setting as default value for omitted settings */
+  unsigned int first_page_column = atoi(strtok(settings, ":"));
```

Comment:
Please use the reentrant version of `strtok`, `strtok_r`, or just use `g_strsplit` as we do elsewhere. You do not have to copy the string if you use the latter.